ci/taos/plugins-base/pr-postbuild-build-tizen.sh

```diff
@@ -92,7 +91,7 @@ function pr-postbuild-build-tizen-run-queue(){
         --define "__ros_verify_enable 1" \
         --define "_pr_start_time ${input_date}" \
         --define "_skip_debug_rpm 1" \
-        --buildroot ./GBS-ROOT/  | tee ../report/build_log_${input_pr}_tizen_$1_output.txt
+        --buildroot ./GBS-ROOT/ 2> ../report/build_log_${input_pr}_tizen_$1_error.txt 1> ../report/build_log_${input_pr}_tizen_$1_output.txt
```

Comment:
Nice catch. FYI,  There are different goals between tee and 0/1/2. 
* **tee** : to archive a display message 
* **0/1/2**:
  * 0: to archive input messages (stdin)
  * 1: to archive output messages (stdout)
  * 2: to archive error messages (stderr) in case of a command is not normally executed.